src/interface/events.ts

```diff
@@ -8,13 +8,15 @@ export enum InterfaceEventName {
   EXPLORE_BANNER_CLICKED = 'Explore Banner Clicked',
   EXPLORE_SEARCH_SELECTED = 'Explore Search Selected',
   EXPLORE_TOKEN_ROW_CLICKED = 'Explore Token Row Clicked',
+  EXTERNAL_LINK_CLICK = 'External Link Click',
   FIAT_ONRAMP_BANNER_CLICKED = 'Fiat Onramp Banner Clicked',
   FIAT_ONRAMP_WIDGET_OPENED = 'Fiat Onramp Widget Opened',
   MINI_PORTFOLIO_TOGGLED = 'Mini Portfolio Drawer Toggled',
   NAVBAR_RESULT_SELECTED = 'Navbar Result Selected',
   NAVBAR_SEARCH_EXITED = 'Navbar Search Exited',
   NAVBAR_SEARCH_SELECTED = 'Navbar Search Selected',
   RISK_CHECKED = 'Risk Checked',
+  SHOW_LEGAL_MODAL = 'Show Legal Modal',
```

Comment:
Can this be a page view event for the modal?

Reply:
sure